src/pages/note/index.tsx

```diff
@@ -48,10 +49,13 @@ const Note = () => {
     }
   }, [senderEmail, senderName, setResponding, setSelectedAnswer]);
 
+
+
   return (
     <>
       {/* <HeartDoodle /> */}
       <OptionsList options={options} />
+     { !responding && <HeardDoodleDialogue /> }
```

Comment:
This condition looks more like what we want to check for - `responding`. It seems to me that we might not need to check any state condition at all in `HeardDoodleDialogue` itself since as long as we want the dialogue to exist, we want it to be able to show a toast as well.